consensus/spos/consensusState.go

```diff
@@ -384,3 +389,78 @@ func (cns *ConsensusState) GetMultikeyRedundancyStepInReason() string {
 func (cns *ConsensusState) ResetRoundsWithoutReceivedMessages(pkBytes []byte, pid core.PeerID) {
 	cns.keysHandler.ResetRoundsWithoutReceivedMessages(pkBytes, pid)
 }
+
+// GetRoundCanceled returns the state of the current round
+func (cns *ConsensusState) GetRoundCanceled() bool {
+	return cns.RoundCanceled
+}
+
+// SetRoundCanceled sets the state of the current round
+func (cns *ConsensusState) SetRoundCanceled(roundCanceled bool) {
+	cns.RoundCanceled = roundCanceled
```

Comment:
we can now unexport the internal fields, todo for the next PRs

Reply:
we can also unexport `ConsensusState`

Reply:
I would unexport maybe just the fields.